lighthouse-core/scripts/run-mocha.sh

```diff
@@ -18,10 +18,13 @@ elif [ "$flag" == '--cli' ]; then
   _runmocha 'lighthouse-cli'
 elif [ "$flag" == '--viewer' ]; then
   _runmocha 'lighthouse-viewer'
+elif [ "$flag" == '--extension' ]; then
```

Comment:
I missed the earlier conversation, but do we want this in the unit tests? Considerably slower than (most? all?) the rest of them. It's also not really a unit test...

`yarn test-extension` or something instead?

Reply:
having it under `test` in the extension `package.json` would make sense as well

Reply:
something as `"unit-extension": "cd lighthouse-extension && yarn test"`?

Reply:
bingo. so remove it from run-mocha. add `test-extension` to root manifest. add `test` to extension manifest. put `test-extension` in travis.yml